packages/cozy-mespapiers-lib/src/helpers/queries.js

```diff
@@ -112,6 +112,20 @@ export const buildConnectorsQueryById = (id, enabled = true) => ({
   }
 })
 
+export const buildKonnectorsQueryByQualificationLabels = (
+  labels,
+  enabled = true
+) => ({
+  definition: Q(KONNECTORS_DOCTYPE)
```

Comment:
If you use enabled, then the definition must be wrapped in a function.  https://github.com/cozy/cozy-contacts/pull/924/files

Reply:
Pour faire plus simple j'ai rajouté le wrapper partout dans un commit